docs/layouts/shortcodes/generated/auto_scaler_configuration.html

```diff
@@ -188,6 +188,12 @@
             <td>Duration</td>
             <td>Time interval to resend the identical event</td>
         </tr>
+        <tr>
+            <td><h5>job.autoscaler.scaling.radical.enabled</h5></td>
```

Comment:
Coming at this cold, its not at all clear to me what radical means. While the description goes some way towards clarifying the intent it doesn't feel like a great term, additionally following through the JIRA links `radical` feels like a very off term for a default (assuming I'm following properly). I wonder if `job.autoscaler.scaling.maximizeUtilisation.enabled` would make things more explicit?